ray-operator/controllers/ray/batchscheduler/yunikorn/yunikorn_scheduler.go

```diff
@@ -0,0 +1,70 @@
+package yunikorn
+
+import (
+	"context"
+	"github.com/go-logr/logr"
+	rayv1 "github.com/ray-project/kuberay/ray-operator/apis/ray/v1"
+	schedulerinterface "github.com/ray-project/kuberay/ray-operator/controllers/ray/batchscheduler/interface"
+	v1 "k8s.io/api/core/v1"
```

Comment:
Use `corev1` instead of `v1` to keep the consistency with the codebase.

Reply:
Fixed.